docs/release-notes/8.0.33-25.md

```diff
@@ -0,0 +1,108 @@
+
+Percona Server for MySQL is trusted by thousands of enterprises to provide better performance and concurrency for their most demanding workloads.
+
+## Release highlights
+
+* This release adds the following MyRocks variables:
+
+  * `rocksdb_bulk_load_use_sst_partitioner`
+
+  * `rocksdb_use_hyper_clock_cache`
+
+  * `rocksdb_converter_record_cached_length`
+
+  * `rocksdb_alter_table_comment_inplace`
+
+  * `rocksdb_charge_memory`
+
+  * `rocksdb_column_default_value_as_expression`
+
+  * `rocksdb_corrupt_data_action`
+
+  * `rocksdb_disable_instant_ddl`
+
+  * `rocksdb_enable_delete_range_for_drop_index`
+
+  * `rocksdb_partial_index_blind_delete`
+
+  * `rocksdb_protection_bytes_per_key`
+
+  * `rocksdb_use_write_buffer_manager`
+
+* This release adds a new MyRocks Information Schema Table `ROCKSDB_LIVE_FILES_METADATA`.
+
+* This release removes `rocksdb_instant_ddl` variable. Use `rocksdb_disable_instant_ddl` instead. 
+
+Improvements and bug fixes introduced by Oracle for MySQL 8.0.33 and included in Percona Server for MySQL are the following:
+
+* The `audit_log_database` server system variable permits specifying other databases in the global schema namespace at server startup. The default setting for table storage is `mysql` system database.
+
+* The `INSTALL COMPONENT` includes the `SET` clause. The `SET` clause sets the values of component system variables when installing one or several components. This reduces the inconvenience and limitations associated with assigning variable values in other ways.
+
+* The mysqlbinlog `--start-position` accepts values up to `18446744073709551615`. If the `--read-from-remote-server` or `--read-from-remote-source` option is used, the maximum is `4294967295`. (Bug #77818, Bug #21498994)
+
+* Using a generated column with `DEFAULT(col_name)` to specify the default value for a named column is not allowed and throws an error message. (Bug #34463652, Bug #34369580)
+
+* Not all possible error states were reported during the binary log recovery process. (Bug #33658850)
+
+* User-defined collations are deprecated. The usage of the following user-defined collations causes a warning that is written to the log:
+
+  * When `COLLATE` is followed by the name of a user-defined collation in an SQL statement.
+
+  * When the name of a user-defined collation is used as the value of `collation_server`, `collation_database`, or `collation_connection`.
+
+The support for user-defined collations will be removed in a future releases of MySQL.
+
+Find the full list of bug fixes and changes in the [MySQL 8.0.33 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-33.html).
+
+## New features
+
+* [PS-8188](https://jira.percona.com/browse/PS-8188): Adds `clone_exclude_plugins_list` option which provides a list of plugins that are not installed on the recipient. 
```

Comment:
The wording is not clear here. By default the list of installed plugins on donor and recipient must match. This 
 variable `clone_exclude_plugins_list` allows to specify the list exceptions.

Reply:
updated